services/drone/drone-build.service.js

```diff
@@ -0,0 +1,131 @@
+'use strict'
+
+const Joi = require('joi')
+const serverSecrets = require('../../lib/server-secrets')
+const { isBuildStatus, renderBuildStatusBadge } = require('../build-status')
+const { optionalUrl } = require('../validators')
+const { BaseJsonService } = require('..')
+
+const DroneBuildSchema = Joi.object({
+  status: Joi.alternatives()
+    .try(isBuildStatus, Joi.equal('none'))
+    .required(),
+}).required()
+
+const queryParamSchema = Joi.object({
+  server: optionalUrl,
+}).required()
+
+module.exports = class DroneBuild extends BaseJsonService {
+  static get category() {
+    return 'build'
+  }
+
+  static get route() {
+    return {
+      queryParamSchema,
+      base: 'drone/build',
+      pattern: ':user/:repo/:branch*',
+    }
+  }
+
+  static get staticPreview() {
+    return {
+      message: 'passing',
+      color: 'brightgreen',
+    }
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'build',
+    }
+  }
+
+  static render({ status }) {
```

Comment:
can also remove this function, since it's no longer being called from anywhere (`handle` is using the `renderBuildStatusBadge` helper function directly which is 👍 )

Reply:
Done!